pages/1.11/networking/DNS/index.md

```diff
@@ -0,0 +1,88 @@
+}
+```
+The `/mygroup/myapp` is running on a virtual network called "dcos". We will use this app to describe the different FQDN it receives from DC/OS DNS.
+
+*NOTE*: While the example uses a Docker container the same FQDNs are generated for a UCR container as well.
+
+# `myapp.mygroup.marathon.mesos`
+For any service launched on DC/OS, every instance of the service would get the FQDN `<service-name>.<group-name>.marathon.mesos`. 
+
+This FQDN is resolved by `mesos-dns`. Please read the "Recommendation" section to understand the difference between using this FQDN vs using an FQDN from the `*.directory` TLD. The actual IP address to which this FQDN maps will depend on the networking mode on which the container is running. For our given example, since the application is running on an SDN the IP address will be the container's IP address. That said, if the application was running on host mode networking, than the IP address would have been that of the agent on which the application instance is running. For bridge mode networking, the IP address would again be that of the agent since the application would be accessible only through D-NAT rules on the agent.
+
+# `myapp-mygroup.marathon.containerip.dcos.thisdcos.directory`
+This FQDN is resolved by `dcos-dns`. For `container` and `bridge` mode networking, this FQDN would resolve to the container's IP address, whereas for `host` mode networking it would resolve to the agent IP, since that is the only IP address visible to the container.
+
+# `myapp-mygroup.marathon.agentip.dcos.thisdcos.directory`
+This FQDN is resolved by `dcos-dns`. Irrespective of the networking the FQDN would always resolve to the agent IP on which this application is running.
+
+# `myapp-mygroup.marathon.autoip.dcos.thisdcos.directory`
+This FQDN is resolved by `dcos-dns`. For `container` mode networking this would resolve to the container's IP address, but for `bridge` and `host` mode networking this would resolve to the agent's IP address. The reason for this discrepancy in the resolver, based on the different networking modes, is the connectivity available to the container in these different modes. In `container` mode networking the container can be reached through the container's IP address, however in bridge mode networking the container can be reached only through D-NAT on the agent and hence only through the agent's IP address. Similarly in host mode networking the container can be reached only through the agent's IP address. As can be seen this FQDN is the most versatile in determining connectivity to the container and should be used by default to connect to a given service. 
+
+# `mygroupmyapp.marathon.l4lb.thisdcos.directory`
+This is a special FQDN that is resolved by `dcos-dns`. This FQDN is primarily used for layer-4 load balancing by `dcos-l4lb` to all instances of this service. The FQDN is generated for a service only when load-balancing is explicitly enabled for the service by specifying the `VIP` label:
+```
```

Comment:
I'm not positive, but I think this will format better with newlines before and after the ` ``` `